### PR TITLE
add style for the filter + fix import

### PR DESCRIPTION
**Pull request by M-Valentin-51**

### CODE REVIEW

File: frontend/src/components/filter/Filter.jsx [hunk 1]
```diff
@@ -2,14 +2,12 @@ import React from "react";
 import "../../../style/filter.css";
 import Fuels from "./Fuels";
 import Buttons from "./Buttons";
-import Brands from "./Brands";
 
 function Filter() {
   return (
-    <div>
+    <div className="contenerFilter">
```
Comment (rocambille) [inline frontend/src/components/filter/Filter.jsx:8]: container, not contener

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
conflicts to resolve

File: frontend/src/App.css [hunk 2]
```diff
@@ -1,10 +1,13 @@
 body {
   margin: 0;
+  height: max-content;
+  background-color: #24242a;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
```
Comment (Justine-Merlin) [inline frontend/src/App.css:10]: Delete blank line.

File: frontend/style/filter.css [hunk 3]
```diff
@@ -1,3 +1,14 @@
+:root{
+    --colorPrimary : #24242a;
+    --colorSecondary : #09ace7;
+}
+.contenerFilter{
```
Comment (Justine-Merlin) [inline frontend/style/filter.css:5]: Classname in your jsx component might be containerFilter.

File: frontend/style/filter.css [hunk 4]
```diff
@@ -6,19 +17,27 @@
...
     list-style: none;
-
+    padding : 0;
 }
-.fuelElement {
-  margin: 0.5rem;
+.fuelElement{
+    margin: 0.5rem;
+    color: var(--colorSecondary);
+
```
Comment (Justine-Merlin) [inline frontend/style/filter.css:40]: Delete blank line.

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

**Pull request merged**